tinygrad/helpers.py

```diff
@@ -280,6 +280,12 @@ class CStruct(ctypes.Structure):
 def init_c_var(ctypes_var, creat_cb): return (creat_cb(ctypes_var), ctypes_var)[1]
 def flat_mv(mv:memoryview): return mv if len(mv) == 0 else mv.cast("B", shape=(mv.nbytes,))
 
+# TODO typeshed stub is missing the stub for memoryview float16 cast (format: Literal['e', '@e']). Remove once support is added.
+# src: https://github.com/python/mypy/blob/v1.13.0/mypy/typeshed/stdlib/builtins.pyi#L769
+def to_legacy_fmt(fmt:str):
```

Comment:
I don't think this is the right way to handle this. Think about what you want to do with this, not a noop helper